client/src/styles/bootstrap_ext/_button.scss

```
@@ -157,6 +174,7 @@ $borderRadius: 1000px;
 
 .btn-rk-pink {
   @extend .btn-rk;
+  --#{$prefix}-btn-active-color: white;
```

```suggestion
  --#{$prefix}btn-active-color: white;
```

Also, I'm not sure the rule is needed.